specification/search/data-plane/Azure.Search/readme.md

```diff
@@ -100,10 +100,21 @@ vararg-properties: >-
   SuggestOptions.orderBy, SuggestOptions.searchFields, SuggestOptions.select
 
 directive:
+    # Enable configuration of RestProxy serializer for DocumentsImpl
```

Comment:
I thought this was for SearchIndexClient -- isn't it in the other file already?

Reply:
Will revert this readme change. And work on merge all changes on the other two back in follow PR